ci/apps/ruby/.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.4.1
```

Comment:
I know you copied this from the demo but think it's worth updating this to use 2.6.3 while we're at it? It's probably trivial for such a simple app.

Reply:
Will do!